### PR TITLE
headshots updated

### PR DESCRIPTION
**Pull request by LukeMccon**
Resized headshots

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -22,7 +22,3 @@
 	
 }
 
-.headshot {
-    width: 250px;
-    height: 250px;
-}
```
Comment (adithyabsk) [inline css/main.css]: Were you able to confirm it wasn't a deployment issue? This seemed to work on my computer as well

Comment (LukeMccon) [reply css/main.css]: Not exactly sure how to check that again because we would have to push to production? I just manually resized the images this time.

Comment (adithyabsk) [reply css/main.css]: image

Comment (adithyabsk) [reply css/main.css]: I just downloaded and checked it and something definitely is wrong.
image

Comment (LukeMccon) [reply css/main.css]: Yeah so after downloading that build to my computer that build worked. Idk what happened. I l can just add the css back in ?

Comment (adithyabsk) [reply css/main.css]: Not sure, I want to understand what went wrong before we go back and forth.

Comment (adithyabsk) [reply css/main.css]: Can you try to isolate the issue first?

Comment (adithyabsk) [reply css/main.css]: I'd start by downloading what was deployed and compare those files.

Comment (LukeMccon) [reply css/main.css]: Yeah, so that build I just accidentally commented a few extra div tags out. Check pull request #14.

Comment (LukeMccon) [reply css/main.css]: The headshots class I added myself so I didn't have to resize the images, this new pull request should work and we don't need that css anymore

**Pull request closed without merging**